### PR TITLE
Mkdir on OSX/Darwin requires `-p` argument before dir

### PR DESCRIPTION
**Pull request by samm-git**

### CODE REVIEW

File: docs/HTTP3.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ notice. It needs to be enabled at build-time.
 Further development and tweaking of the HTTP/3 support in curl will happen in
 in the master branch using pull-requests, just like ordinary changes.
 
+To build on MacOS with provided recipes you may want to set `alias nproc="sysctl -n hw.ncpu"`
+before running make commands.
```
Comment (bagder) [inline docs/HTTP3.md]: I propose we drop the `-j` use all together instead to make the command lines work better on more places.

Comment (samm-git) [reply docs/HTTP3.md]: May be we should add a note that parallel builds are supported with a `-j <numcpu>` keyword, so maintainers will use that in their build recipes?

Comment (samm-git) [reply docs/HTTP3.md]: P.S. Just checked `sysctl -n hw.ncpu` also working on the FreeBSD, so it seems that with ncpu for the Linux and alias for the *BSD/Darwin we are covering most of the systems.

Comment (bagder) [reply docs/HTTP3.md]: I don't think we need to promote "make tips" here and maintainers should really already know about `make -j` ...

**Pull request closed without merging**